BUG: Fix version check error for Pandas release candidates.

nkeim opened the pull request:
Discovered while investigating #282. Apprently `StrictVersion` is… strict.

caspervdw commented:
:+1: It might make sense to move the bit of code to the top, so make a global `is_recent_pandas`.

nkeim commented:
Good idea. The test also contained a version check; I've now fixed the code there also. And since the test was in two places, I moved it to `utils.py`.

tacaswell commented on trackpy/utils.py:
```diff
@@ -16,6 +17,14 @@
+try:
+    is_pandas_recent = (StrictVersion(pd.__version__) >=
+                        StrictVersion('0.16.0'))
+except ValueError:  # Probably a release candidate
```
where does the valueerror come from?

nkeim replied:
From the "rc" in the version string. We could use `LooseVersion`, which allows alphabetic characters. But I figured that the test needed a fallback anyhow, so I went for that.

danielballan commented:
Good enough!
